src/render/html_render.py

```diff
@@ -2,8 +2,60 @@
     HorizontalRule, ListParagraph, QuoteParagraph, BoldBlock, ItalicBlock, \
     ImgBlock, LinkBlock, CodeBlock, FencedCodeBlock, StrikethroughBlock, ListWrapper
 
+
 def render(root):
-    return _dfs(root)
+    rendered_child = []
+    for paragraph in root.children:
+        rendered_child.append(_render_a_paragraph(paragraph))
+    return "<html>{}</html>".format("\n".join(rendered_child))
+
+def _render_a_paragraph(paragraph):
+    if isinstance(paragraph, HorizontalRule):
+        return "<hr/>"
+    if isinstance(paragraph, FencedCodeBlock):
+        return "<code>{}</code>".format(paragraph.content())
+    if isinstance(paragraph, HeaderParagraph):
+        return "<h{0}>{1}<h{0}>".format(paragraph.level(), paragraph.content())
+
+    _format = ""
+    if isinstance(paragraph, TextParagraph):
+        _format = "<p>{}</p>"
+    if isinstance(paragraph, ListWrapper):
+        if paragraph.is_ordered():
+            _format = "<ol>\n{}</ol>"
+        else:
+            _format = "<ul>\n{}</ul>"
+    if isinstance(paragraph, QuoteParagraph):
+        _format = "<blockquote>{}</blockquote>"
+    rendered_content = []
+    for element in paragraph.children:
+        rendered_content.append(_render_a_element(element))
+    return _format.format("".join(rendered_content))
+
+
+def _render_a_element(element):
+    _format = ""
+    if isinstance(element, TextBlock):
+        return element.content()
+    if isinstance(element, ImgBlock):
+        return "<img src={} alt={}/>".format(element.url(), element.content())
+    if isinstance(element, LinkBlock):
+        return "<a herf={} />".format(element.content())
```

Comment:
It seems Link should have both content and url 
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links